statsmodels/regression/recursive_ls.py

```diff
@@ -370,10 +398,21 @@ def cusum_squares(self):
            Series B (Methodological) 37 (2): 149-92.
 
         """
-        numer = np.cumsum(self.resid_recursive[self.loglikelihood_burn:]**2)
+        d = max(self.nobs_diffuse, self.loglikelihood_burn)
+        numer = np.cumsum(self.resid_recursive[d:]**2)
         denom = numer[-1]
         return numer / denom
 
+    @cache_readonly
+    def llf_obs(self):
+        from scipy.stats import norm
+        return np.log(norm.pdf(self.resid_recursive, loc=0,
+                               scale=self.filter_results.obs_cov[0, 0]**0.5))
+
+    @cache_readonly
+    def llf(self):
+        return np.sum(self.llf_obs)
```

Comment:
Does this have a standard MLE interpretation?
e.g. would be use this for comparing models, or for aic, bic

Reply:
Yes, this one (computed from the recursive residuals) is the same as the OLS profile log-likelihood function (i.e after concentrating out the scale), the same as you would get from `sm.OLS.loglike` with `scale=None`.